specs/driver_spec.rb

```diff
@@ -55,10 +55,27 @@
     end
   end
 
+  describe "driver total revenue" do
+    before do
+      pass = RideShare::Passenger.new(id: 1, name: "Ada", phone: "412-432-7640")
+      @driver = RideShare::Driver.new(id: 3, name: "Lovelace", vin: "12345678912345678")
+      @trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: pass, start_time: Time.new("2016-08-08"),cost:3.50, end_time: Time.new("2016-08-09"), rating: 5})
+      @driver.add_trip(@trip)
+    end
+
+    it "calculates driver total revenue" do
```

Comment:
You should also have a test for what the total revenue is when there are no trips, and with several trips.